indico/modules/events/tracks/blueprint.py

```diff
@@ -23,6 +25,11 @@
 _bp.add_url_rule('/manage/tracks/<int:track_id>', 'edit_track', RHEditTrack, methods=('GET', 'POST'))
 _bp.add_url_rule('/manage/tracks/<int:track_id>', 'delete_track', RHDeleteTrack, methods=('DELETE',))
 
+_bp.add_url_rule('/manage/track-groups', 'track_groups', RHTrackGroups)
+_bp.add_url_rule('/manage/track-groups/create', 'create_track_group', RHCreateTrackGroup, methods=('GET', 'POST'))
+_bp.add_url_rule('/manage/track-groups/<int:group_id>', 'edit_track_group', RHEditTrackGroup, methods=('GET', 'POST'))
```

Comment:
Are we going with `group_id` or `track_group_id`? I think I've noticed it written differently in some parts.

Reply:
Should be `track_group_id`, let me fix that ;)

Reply:
Updated and thanks for noticing that!